profiles/kentik_snmp/cisco/cisco-all-devices.yml

```diff
@@ -671,3 +671,142 @@ metrics:
           OID: 1.3.6.1.4.1.9.9.13.1.5.1.2
           name: ciscoEnvMonSupplyStatusDescr
         tag: power_status_description
+  - MIB: CISCO-RTTMON-MIB
+    table:
+      OID: 1.3.6.1.4.1.9.9.42.1.2.10
+      name: rttMonLatestRttOperTable
+    symbols:
+      - OID: 1.3.6.1.4.1.9.9.42.1.2.10.1.1
+        name: rttMonLatestRttOperCompletionTime
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.9.9.42.1.2.1.1.3
+          name: rttMonCtrlAdminTag
+        tag: rtt_operation_tag
+      - column:
+          OID: 1.3.6.1.4.1.9.9.42.1.2.1.1.4
+          name: rttMonCtrlAdminRttType
+          enum:
+            echo: 1
+            pathEcho: 2
+            fileIO: 3
+            script: 4
+            udpEcho: 5
+            tcpConnect: 6
+            http: 7
+            dns: 8
+            jitter: 9
+            dlsw: 10
+            dhcp: 11
+            ftp: 12
+            voip: 13
+            rtp: 14
+            lspGroup: 15
+            icmpjitter: 16
+            lspPing: 17
+            lspTrace: 18
+            ethernetPing: 19
+            ethernetJitter: 20
+            lspPingPseudowire: 21
+            video: 22
+            y1731Delay: 23
+            y1731Loss: 24
+            mcastJitter: 25
+            fabricPathEcho: 26
+        tag: rtt_operation_type
+      - column:
+          OID: 1.3.6.1.4.1.9.9.42.1.2.1.1.9
+          name: rttMonCtrlAdminStatus
+          enum:
+            active: 1
+            notInService: 2
+            notReady: 3
+            createAndGo: 4
+            createAndWait: 5
+            destroy: 6
+        tag: rtt_operation_admin_status
+      - column:
+          OID: 1.3.6.1.4.1.9.9.42.1.2.2.1.1
+          name: rttMonEchoAdminProtocol
+          enum:
+            notApplicable(1), ipIcmpEcho(2), ipUdpEchoAppl(3), snaRUEcho(4), snaLU0EchoAppl(5), snaLU2EchoAppl(6), snaLU62Echo(7), snaLU62EchoAppl(8), appleTalkEcho(9), appleTalkEchoAppl(10), decNetEcho(11), decNetEchoAppl(12), ipxEcho(13), ipxEchoAppl(14), isoClnsEcho(15), isoClnsEchoAppl(16), vinesEcho(17), vinesEchoAppl(18), xnsEcho(19), xnsEchoAppl(20), apolloEcho(21), apolloEchoAppl(22), netbiosEchoAppl(23), ipTcpConn(24), httpAppl(25), dnsAppl(26), jitterAppl(27), dlswAppl(28), dhcpAppl(29), ftpAppl(30), mplsLspPingAppl(31), voipAppl(32), rtpAppl(33), icmpJitterAppl(34), ethernetPingAppl(35), ethernetJitterAppl(36), videoAppl(37), y1731dmm(38), y17311dm(39), y1731lmm(40), mcastJitterAppl(41), y1731slm(42), y1731dmmv1(43), fabricPathEchoAppl(44)
```

Comment:
@Mesverrum -  I don't think this syntax is going to work

Reply:
check